src/arena/Details.tsx

```diff
@@ -0,0 +1,76 @@
+import { Dispatch, SetStateAction } from "react";
+import Button from "react-bootstrap/lib/Button";
+
+interface Props {
+  displayDay: number;
+  missions: any[];
+  admin: boolean;
+  status: string;
+  setDisplayDay: Dispatch<SetStateAction<number>>;
+  setStatus: Dispatch<SetStateAction<string>>;
+}
+
+/**
+ * The details component is used to display the details days of the missions.
+ * @param displayDay index of the day to display
+ * @param {Array} missions array of missions
+ * @param admin whether the user is an admin
+ * @param status current status
+ * @function setDisplayDay
+ * @function setStatus
+ */
+function Details({
+  displayDay,
+  missions,
+  admin,
+  status,
+  setDisplayDay,
+  setStatus,
+}: Props) {
+  return (
+    <details>
+      <summary>Time Travel</summary>
+      <div className="options">
+        <p>
+          <b>Currently Day {displayDay + 1}</b>
+        </p>
+        <p>
+          Click a button below to go back to a specific day and add an update
+          you forgot.
+        </p>
+
+        <div className="options-buttons">
+          {missions.map((mission, idx) => (
+            <Button
+              onClick={() => {
+                setDisplayDay(idx);
+              }}
+              // eslint-disable-next-line react/no-array-index-key
+              key={idx}
+            >
+              Day {idx + 1}
+            </Button>
+          ))}
+        </div>
+        <div className="flex">
+          {admin === true ? (
+            <div>
```

Comment:
Can you review the PR that was recently merged in from @annovo - https://github.com/mikhael28/paretOS/pull/195 ? She refactored this part of the components, can you pull in the use of Chips and removing unnecessary divs? If you pull from main, you may get some nasty merge conflicts - save your progress in Git!

Reply:
Seen. I still maintain the same Chips component but I feel like they could have been buttons component directly so that whoever is working on it is going to know that they're button sat a glance.

Reply:
Thank you, I agree - it could go either way, chips vs button. We can always revisit later if, for accessibility purposes, one solution is better than the other.